src/mc.rs

```diff
@@ -1305,11 +1306,19 @@ impl warp8x8t::Fn {
         my: c_int,
         bd: BD,
     ) {
-        let tmp = tmp.as_mut_ptr();
-        let tmp_stride = tmp_stride as isize;
         let src = src.cast();
         let bd = bd.into_c();
-        self.get()(tmp, tmp_stride, src, src_stride, abcd, mx, my, bd)
+        self.get()(
+            tmp.as_mut_ptr(),
```

Comment:
Btw, I've been preferring putting casts, etc. on their own lines as `let`s in these wrapper methods.  Just slightly easier to read IMO.